content/news/posts/2023/contributor-interview-louis-authie.md

```diff
@@ -0,0 +1,96 @@
+Thirdly, contributing gives you some sort of authority: the more you contribute, the more you get to know the PrestaShop community. And the more you know contributors, the more inclined they’ll be to help you.
+
+**Do you have any advice for first-time contributors?**
+
+I believe there’s no small contribution, you can start [contributing right away](https://docs.prestashop-project.org/v.8-documentation/v/english/). It could be just fixing a typo, or changing colors on docs. Of course, it’s easy to do, but it’s still already a big step.
+
+**How do you hope to see the PrestaShop project evolve in 2023?**
+
+From what I can see, PrestaShop is moving quickly towards newer versions (v.8), which is good. When the project finally transitions to being 100% Symfony, it’ll be of great help to the community. I feel that the migration period left a lot of people behind as it’s hard to keep up with everything.
+Once everything is unified, I believe we’ll be able to keep up against other CMS. It’ll also allow us to upgrade from Symfony 4.4 to 6.2 so we can take care of real issues, focused on merchants’ needs. 
+
+From my point of view, we spent the migration period moving to something more sustainable in the long run. Of course, we built and gathered a lot of tools allowing us to work faster, but we left some important features behind. In the end, I believe the project should focus on more merchant-oriented features such as product packs or bulk discounts, which constitute a big part of merchant demands.
+
+Overall, I’m sure PrestaShop 9 will strengthen the use of Symfony, which can only be good for everyone.
+
+**Is there anything else you want to add?**
+
+Yes! I’ve been writing a book for the last 6 months: *Practical Module Development for PrestaShop 8*!
+
+I asked myself a lot of questions about the migration but I didn’t find a lot of answers. I know I can find some answers in the dev doc as it has evolved over the last few years, but it’s still a listing of features. It won’t help me much if I want to start developing things. If I want to make a module, where can I find resources? There are a few pages here and there on GitHub but it was not easy to find things that could help me.
+
+So I took all of my contributions and knowledge and decided to share it with others. Around that same time, Packt, my publisher, got in touch with me. Originally, I was supposed to write it with Fabien Serny (a former PrestaShop developer who wrote a book on the same topic in 2014), but he didn’t have time so I took it upon myself to write it. 
+
+My book should be of great help to the community. It answers the most commonly asked questions in module development.
+I’m not promoting my book for money: all royalties and benefits will go to FOP.
+
+If you buy the book on Amazon and use the voucher **20LOUIS**, you get 20% off, here: https://www.amazon.com/gp/mpc/A1C6DNDDRD3ZU5. The voucher’s active until May 31st.
```

Comment:
I was questioning whether it was a good thing to add a link to his book on this interview. But, the fact he is promoting a book where revenue goes to a non-profit like FOP - not in his pocket - feels like it's okay to add his link.